src/main/scala/org/virtuslab/internal/load/reader/StateStack.scala

```diff
@@ -0,0 +1,23 @@
+package org.virtuslab.internal.load.reader
+
+package org.virtuslab.internal.load.reader
+
+case class StateStack(var stack: List[ReaderState]) {
```

Comment:
There are too many brackets in this case class 😄
For me, mutability is also ok as it's just a stack.

Reply:
I removed the unnecessary brackets